controllers/datadogagent/feature/ids.go

```diff
@@ -23,6 +23,8 @@ const (
 	LiveContainerIDType = "live_container"
 	// LiveProcessIDType Live Process feature.
 	LiveProcessIDType = "live_process"
+	// ProcessDiscoveryIdType Process Discovery feature.
+	ProcessDiscoveryIdType = "process_discovery"
```

Comment:
```suggestion
	ProcessDiscoveryIDType = "process_discovery"
```
maybe eventually we'll switch to `Id` 😅 , but for consistency's sake